lib/client.js

```diff
@@ -378,6 +379,19 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
+
+    async checkEventJornal(journalUrl) {
+        const ioEvents = new AdobeIOEvents({
+            accessToken: this.accessToken,
+            orgId: this.integration.technicalAccount.org
+        });
+        try {
+            await ioEvents.getEventsFromJournal(journalUrl);
+        } catch(error) {
+            console.error(error);
```

Comment:
Is the log here only for debugging? Because if so, you could remove the try/catch and let it throw.